### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by kravien**
### Summary
Application work has been distributed between 2 servers.  

First server handles HTTP requests for `index.html` and `form.html`, while the second one handles `charge.html` and `auth.html`. 

Basic application concepts left unchanged.
### How to Verify 
See README.md
------------

### CODE REVIEW

File: apps/server2/server.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+from urllib.parse import parse_qs
+from http.cookies import SimpleCookie
+
+import helpers
+
+
+class SecondServer(helpers.BaseServer):
```
Comment (andreybulygin) [inline apps/server2/server.py]: Please do:
1. move most of (90+%) logic into the `BaseServer` for both of your servers, no worries about conditionally unneeded/unused code.
1. make your servers (or just the first server if it's enough) configurable based on the current settings

File: apps/server1/server.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from http.cookies import SimpleCookie
+
+import helpers
+from apps.server2.settings import HOST as HOST2, PORT as PORT2
```
Comment (andreybulygin) [inline apps/server1/server.py]: fair enough, but semantically better to rename those in smth like `CHARGE_HOST/CHARGE_PORT`

File: apps/server2/server.py (hunk 1 above)
Comment (kravien) [reply apps/server2/server.py]: @andreybulygin , what should I leave then in each of the servers, if most of logic would be in `BaseServer`? And it's not clear to me what you mean in the second remark.

Comment (andreybulygin) [reply apps/server2/server.py]: @kravien trying to summarize both points: depending on whether you need any configurational changes inside concrete classes, you might not need those classes at all, and you should be good to use the base server for both directly (only rename it to a concrete then).

Comment (kravien) [reply apps/server2/server.py]: fixed

File: apps/server1/server.py (hunk 2 above)
Comment (kravien) [reply apps/server1/server.py]: fixed

File: templates/auth.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <meta charset="UTF-8">
+    <title>{procedure}</title>
+</head>
+<body style="font-family: 'Roboto Light', sans-serif; font-size: 16pt; color:#9e3f3f">
+<br/>
+    <div style="text-align: center;">
+        {message}
+    <p>
+        <br/>
+        <a href="http://localhost:8001/form">
```
Comment (andreybulygin) [inline templates/auth.html]: Such hardcodes should ideally be replaced too.

Comment (kravien) [reply templates/auth.html]: fixed

Comment (andreybulygin) [reply templates/auth.html]: 👍

**Pull request closed without merging**